components/raftstore/src/store/fsm/peer.rs

```diff
@@ -692,7 +700,8 @@ where
             self.propose_pending_batch_raft_command();
         } else if self.fsm.batch_req_builder.has_proposed_cb
             && self.fsm.batch_req_builder.propose_checked.is_none()
-            && let Some(cmd) = self.fsm.batch_req_builder.request.take()
+            &&
```

Comment:
It looks a bit weird to me that `&&` alone occupies a whole line.

Reply:
Fixed